__tests__/data-residency.test.js

```diff
@@ -0,0 +1,134 @@
+import { resolveDataPlaneUrl } from '../utils/utils';
+import { DEFAULT_DATAPLANE_URL } from '../utils/constants';
+
+const sourceConfigWithBothDataPlane = {
+  source: {
+    dataplanes: {
+      US: [
+        {
+          url: 'https://sample.rudderlabs.com/us',
+          default: true,
+        },
+      ],
+      EU: [
+        {
+          url: 'https://sample.rudderlabs.com/eu',
+          default: true,
```

Comment:
How can there be two defaults?

Reply:
Yes there can be. Here is the PRD link, check the source config response.